group_variables.go

```diff
@@ -38,6 +38,7 @@ type GroupVariable struct {
 	Key       string `json:"key"`
 	Value     string `json:"value"`
 	Protected bool   `json:"protected"`
+	Masked    bool   `json:"masked"`
```

Comment:
I cannot seem to find this field in the docs for group variables. Are you sure its part of the response data?

Reply:
your right! it does not look like its available for group vars. I will change the code accordingly.